.github/workflows/README.md

```diff
@@ -0,0 +1,59 @@
+# GitHub Actions Workflows
+
+## Setup
+- workloads run using [GitHub self-hosted runners](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-self-hosted-runners)
+- project admins maintain a private Google Compute Engine VM for running tests
+  - VM should be at least n1-standard-4 with 50GB persistent disk
+  - instructions for setting up the VM can be found in repo settings under "Actions"
```

Comment:
it's worth noting that the VM should be run with an empty (permissionless) service account, otherwise anyone who can send a PR can steal an `Editor` token for project.

Reply:
good point. added